src/patito/_pydantic/dtypes/utils.py

```diff
@@ -15,12 +15,13 @@
 )
 
 import polars as pl
-from polars.datatypes import DataType, DataTypeClass, DataTypeGroup, convert
-from polars.datatypes.constants import (
+from polars.datatypes import DataType, DataTypeClass, convert
+from polars.datatypes.group import (
```

Comment:
I'd really caution against using these at all. I might just rename this module to `_group` before 1.0 to really make that clear.

What are the groups really used for?

Reply:
It's for all the type checking from python types to pydantic and then polars afaik